doc/contributor/using-eclipse.md

```diff
@@ -41,9 +43,32 @@ There should be now 4 projects in your workspace:
 * `TRUFFLERUBY-TEST.dist`
 * `TRUFFLERUBY.dist`
 
-## Running from the Eclipse Files Directly
+### Running from the Eclipse Files Directly
 
 ```bash
 jt ruby -e 'p Truffle'
 Truffle
 ```
+
+## Advanced Setup
+
+**Note:** This setup is not widely tested.
+
+When building using specific env files, with `JT_ENV` or `--env`, mx will
+need to be told about it explicitly. For instance:
+
+```bash
+jt mx --env jvm-ce eclipseinit
+```
+
+When working on the Graal/Truffle code itself, the following two lines will
+configure the build so that incremental compilation and even hot code
+replacement works:
+
+```bash
+MX_BUILD_EXPLODED=true
+LINKY_LAYOUT=*.jar
```

Comment:
Should these two lines use `export`? Do you set them in some shell profile, or in a mx env file?

Reply:
I rephrased the paragraph before to explicitly point to the env file.